packages/web-components/src/components/va-alert/test/va-alert.e2e.ts

```diff
@@ -13,7 +13,7 @@ describe('va-alert', () => {
         <mock:shadow-root>
           <div class="alert info">
             <i aria-hidden="true" role="img"></i>
-            <div class="body">
+            <div class="body" role="presentation">
```

Comment:
This fixes the issue from the component-library side of things. Any React applications that consume this component will need to be updated with role="presentation" to prevent NVDA from announcing the element as clickable.